### DISCUSSION TITLE
that's not the website

### DISCUSSION THREAD

**Discussion by tyboxsmash** [Fuwn/monkey-type-desktop#2] [General]
the website is https://monkeytype.com/ , not https://monkey-type.com/ this
that's why it's not working rn

**Comment by Fuwn** [comment]
@tyboxsmash, this repository is not being actively maintained anymore. I'll fix it anyway though, thanks for the tip.

**Comment by Fuwn** [comment]
Never mind, I just realized that this issue is actually irrelevant. We've been using the updated domain since May 17th...

![](https://strelizia.cc/Fb3TsFv0zUEPtOyQtiWk6DsnEqpSPhJx.png)
